public/app/features/dashboard-scene/scene/DashboardControls.tsx

```diff
@@ -240,6 +234,8 @@ function getStyles(theme: GrafanaTheme2) {
       display: 'flex',
       justifyContent: 'flex-end',
       gap: theme.spacing(1),
+      marginBottom: theme.spacing(1),
+      float: 'right',
```

Comment:
@oscarkilhed this float is making the controls not take up the height the it needs, so if you only have time controls (no variables), the spacing between time picker and the dashboard panels is wrong